packages/react-core/src/components/DatePicker/examples/DatePickerBasic.tsx

```diff
@@ -4,8 +4,8 @@ import { DatePicker } from '@patternfly/react-core';
 export const DatePickerBasic: React.FunctionComponent = () => (
   <DatePicker
     // eslint-disable-next-line no-console
-    onBlur={(str, date) => console.log('onBlur', str, date)}
+    onBlur={(event, str, date) => console.log('onBlur', str, date)}
```

Comment:
Since other examples have done it, it'd be worth prefacing the `event` arg in these example files with an underscore since they're not being utilized.